deployment/aws-content-localization-use-existing-mie-stack.yaml

```diff
@@ -157,20 +143,14 @@ Resources:
         IdentityPoolId: !GetAtt ContentAnalysisAuthStack.Outputs.IdentityPoolId
         PoolClientId: !GetAtt ContentAnalysisAuthStack.Outputs.UserPoolClientId
 
-  
-
-
 Outputs:
-  ContentLocalizationSolution:
+  CloudfrontUrl:
```

Comment:
I prefer the original name in this case.

Reply:
Please change this back to ContentLocalizationSolution

Reply:
done